public/assets/js/main.js

```diff
@@ -0,0 +1,125 @@
+/*=============== SHOW MENU ===============*/
+const navMenu = document.getElementById('nav-menu'),
+  navToggle = document.getElementById('nav-toggle'),
```

Comment:
This JS file can be removed. since every thing going to be in react

Reply:
I have not applied few things in react right now , so i have kept this file . Will do it afterwards mostly in the next commit . 👍

Reply:
I'm guessing you copy pasted the whole thing from your html css js code to this, try not to do that or at least remove unused files before pushing it onto github, you could even add it to .gitignore then push